core/runtime/pom.xml

```diff
@@ -120,6 +120,12 @@
                         <parentFirstArtifact>org.objenesis:objenesis</parentFirstArtifact>
                         <parentFirstArtifact>net.bytebuddy:byte-buddy</parentFirstArtifact>
                         <parentFirstArtifact>net.bytebuddy:byte-buddy-agent</parentFirstArtifact>
+                        <parentFirstArtifact>org.junit.jupiter:junit-jupiter-api</parentFirstArtifact>
+                        <parentFirstArtifact>org.junit.jupiter:junit-jupiter</parentFirstArtifact>
```

Comment:
This is just a pom.xml depending on `api`, `params` and `engine` to save users from adding them explicitly in their poms. Is it necessary here?

Reply:
In that case no.